### ISSUE TITLE
Single line comments block first line of code

### ISSUE THREAD

**Issue by MilesCranmer** [MilesCranmer/gso#23]
E.g., for https://stackoverflow.com/a/15978154/2689923, the first line of each code block gets commented out by the make comment character, "`#`":

```
 39 #Now for building unistd_32.h:                                                  
 40 #F1_EXISTS=$(shell [ -e /usr/include/asm/unistd_32.h ] && echo 1 || echo 0 )    
 41 ifeq ($(F1_EXISTS), 1)                                                          
 42     $(info Copying custom header)                                               
 43     $(shell sed -e 's/__NR_/__NR32_/g' /usr/include/asm/unistd_32.h > unistd_32.h)
 44 else                                                                            
 45     F2_EXISTS=$(shell [[ -e /usr/include/asm-i386/unistd.h ]] && echo 1 || echo 0 )
 46     ifeq ($(F2_EXISTS), 1)                                                      
 47         $(info Copying custom header)                                           
 48         $(shell sed -e 's/__NR_/__NR32_/g' /usr/include/asm-i386/unistd.h > unistd_32.h)
 49     else                                                                        
 50         $(error asm/unistd_32.h and asm-386/unistd.h does not exist)            
 51     endif                                                                       
 52 endif 
```

This isn't a fatal bug, it's just annoying. I'll fix it soon.

**Comment by MilesCranmer** [comment]
Fixed with 00b471efd48d09ed1a542c2b6f5c88033e7f5f3a.

**Comment by MilesCranmer** [comment]
Closing with #24
